conans/test/generators/make_test.py

```diff
@@ -0,0 +1,320 @@
+
+        inc1 = os.path.join(tmp_folder1, 'include1').replace('\\', '/')
+        inc2 = os.path.join(tmp_folder2, 'include2').replace('\\', '/')
+        self.assertIn("CONAN_INCLUDE_PATHS +=  \\\n"
+                      "$(CONAN_INCLUDE_PATHS_MYPKG1) \\\n"
+                      "$(CONAN_INCLUDE_PATHS_MYPKG2)", content)
+        self.assertIn("CONAN_INCLUDE_PATHS_MYPKG1 +=  \\\n%s" % inc1, content)
+        self.assertIn("CONAN_INCLUDE_PATHS_MYPKG2 +=  \\\n%s" % inc2, content)
+
+        lib1 = os.path.join(tmp_folder1, 'lib1').replace('\\', '/')
+        lib2 = os.path.join(tmp_folder2, 'lib2').replace('\\', '/')
+        self.assertIn("CONAN_LIB_PATHS +=  \\\n"
+                      "$(CONAN_LIB_PATHS_MYPKG1) \\\n"
+                      "$(CONAN_LIB_PATHS_MYPKG2)", content)
+        self.assertIn("CONAN_LIB_PATHS_MYPKG1 +=  \\\n%s" % lib1, content)
+        self.assertIn("CONAN_LIB_PATHS_MYPKG2 +=  \\\n%s" % lib2, content)
+
+        bin1 = os.path.join(tmp_folder1, 'bin1').replace('\\', '/')
+        bin2 = os.path.join(tmp_folder2, 'bin2').replace('\\', '/')
+        self.assertIn("CONAN_BIN_PATHS +=  \\\n"
+                      "$(CONAN_BIN_PATHS_MYPKG1) \\\n"
+                      "$(CONAN_BIN_PATHS_MYPKG2)", content)
+        self.assertIn("CONAN_BIN_PATHS_MYPKG1 +=  \\\n%s" % bin1, content)
+        self.assertIn("CONAN_BIN_PATHS_MYPKG2 +=  \\\n%s" % bin2, content)
+
+        self.assertIn("CONAN_LIBS +=  \\\n$(CONAN_LIBS_MYPKG1) \\\n$(CONAN_LIBS_MYPKG2)", content)
+        self.assertIn("CONAN_LIBS_MYPKG1 +=  \\\nlibfoo", content)
+        self.assertIn("CONAN_LIBS_MYPKG2 +=  \\\nlibbar", content)
+
+        self.assertIn("CONAN_CFLAGS +=  \\\n$(CONAN_CFLAGS_MYPKG1) \\\n$(CONAN_CFLAGS_MYPKG2)",
+                      content)
+        self.assertIn("CONAN_CFLAGS_MYPKG1 +=  \\\n-fPIC", content)
+        self.assertIn("CONAN_CFLAGS_MYPKG2 +=  \\\n-mtune=native", content)
+
+        self.assertIn("CONAN_CPPFLAGS +=  \\\n$(CONAN_CPPFLAGS_MYPKG1) \\\n$(CONAN_CPPFLAGS_MYPKG2)",
+                      content)
+        self.assertIn("CONAN_CPPFLAGS_MYPKG1 +=  \\\n-fPIE", content)
+        self.assertIn("CONAN_CPPFLAGS_MYPKG2 +=  \\\n-march=native", content)
+
+        self.assertIn("CONAN_SHAREDLINKFLAGS +=  \\\n"
+                      "$(CONAN_SHAREDLINKFLAGS_MYPKG1) \\\n"
+                      "$(CONAN_SHAREDLINKFLAGS_MYPKG2)", content)
+        self.assertIn("CONAN_SHAREDLINKFLAGS_MYPKG1 +=  \\\n-framework Cocoa", content)
+        self.assertIn("CONAN_SHAREDLINKFLAGS_MYPKG2 +=  \\\n-framework AudioFoundation", content)
+
+        self.assertIn("CONAN_EXELINKFLAGS +=  \\\n"
+                      "$(CONAN_EXELINKFLAGS_MYPKG1) \\\n"
+                      "$(CONAN_EXELINKFLAGS_MYPKG2)", content)
+        self.assertIn("CONAN_EXELINKFLAGS_MYPKG1 +=  \\\n-framework QuartzCore", content)
+        self.assertIn("CONAN_EXELINKFLAGS_MYPKG2 +=  \\\n-framework VideoToolbox", content)
```

Comment:
I prefer to see all the file written and compare with it... all these previous _asserts_ are checking for almost it all, aren't they?

Reply:
ok, I could change it. Took it from the premake test bu I agree it would be easier to read

Reply:
i was thinking about this at end of day yesterday.  Please feel free to refactor the operators in any way that makes sense.  Looking at the current state of the generator, it's possible you can just inline the "assignment" operator text wherever it's used and eliminate the two properties altogether.